api/services/administration/general/responses/getAccountUsers.yaml

```diff
@@ -64,8 +64,24 @@ content:
                       description: >
                         Array of data objects
                       items:
-                        - $ref: "../schemas/getAccountUsers-example1.yaml"
-                        - $ref: "../schemas/getAccountUsers-example2.yaml"
+                        anyOf:
+                          - $ref: "../schemas/getAccountUsers-example1.yaml"
+                          - $ref: "../schemas/getAccountUsers-example2.yaml"
+                      example:
+                        - mappedAccountIds:
+                          - act-0b18f123
+                          - act-0b18f124
+                          - act-0b18f125
+                          userId: u-19265
+                          displayName: test
+                          email: test@spot.io
+                        - mappedAccountIds:
+                          - act-0b18f123
+                          - act-0b18f124
+                          - act-0b18f125
+                          userId: u-19266
+                          displayName: test
+                          email: abc@google.com
```

Comment:
Can you explain this change please? It seems like proper examples are already present here:
https://docs.spot.io/api/#operation/GetAccountsUsers

Reply:
The examples are indeed properly displayed right now, but that is since the examples are directly under "items" with the $refs themselves instead of being under an anyOf/allOf/oneOf object, which is a semantic error (many such errors were handled in this pull request).

Once you add both examples under such a anyOf/allOf/oneOf object, only one example is displayed (in most cases). Therefore I added the examples manually (this was the solution in many other similar cases).